amundsen_application/api/utils/metadata_utils.py

```diff
@@ -150,16 +150,75 @@ def marshall_dashboard_full(dashboard_dict: Dict) -> Dict:
     return dashboard_dict
 
 
-def _update_prog_descriptions(prog_descriptions: List) -> None:
+def _update_prog_descriptions(prog_descriptions: List) -> Dict:
     # We want to make sure there is a display title that is just source
     for desc in prog_descriptions:
         source = desc.get('source')
         if not source:
             logging.warning("no source found in: " + str(desc))
     prog_display_config = app.config['PROGRAMMATIC_DISPLAY']
+
+    # TEST DATA
+    """
+    PROGRAMMATIC_DISPLAY = {
+       'RIGHT': {
+         "test8" : {},
+         "test9" : {},
+         "test7" : { "display_order": 0 }
+       },
+       'LEFT': {
+         "test3": {},
+         "test1" : { "display_order": 1 },
+         "test0" : { "display_order": 0 },
+         "test2" : { "display_order": 2 }
+       },
+       'test5': {"display_order": 1},
+       'test4': {"display_order": 0},
+    }
+    """
+    prog_descriptions = []
+    prog_descriptions.append({'source': 'test7', 'text': 'Test'})
```

Comment:
I assume this is for testing?

Reply:
Yes, and setting up different edge cases.